components/stacked-vertical-bar-chart/styles.css

```diff
@@ -4,21 +4,27 @@
   display: flex;
   align-items: stretch;
   width: 100%;
-  height: 2rem;
 }
 
 .bar {
-  display: flex;
   flex: 1;
-  align-items: center;
-  justify-content: space-between;
+  height: 2rem;
   padding-right: 0.5rem;
   padding-left: 0.5rem;
   margin: 0;
   overflow: hidden;
+  text-align: right;
 }
 
 .bar > span {
+  display: inline-block;
+  padding: 0.25rem;
```

Comment:
This may break If the line-height changes globally. I recommend to explicitly set `line-height: 1` here and change current padding to `0.5rem`.